action.yml

```diff
@@ -130,6 +151,16 @@ runs:
         curl -sf https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | DESIRED_VERSION="${{ inputs.helm-version }}" bash
       shell: bash
 
+    - name: Set version output
+      id: set-output
+      run: |
+        echo "::set-output name=kubeconfig::$HOME/.kube/config"
+        echo "::set-output name=k3s-version::$(k3s --version | sed 's/.*\(v[0-9][^ ]*\).*/\1/')"
+        echo "::set-output name=k8s-version::$(k3s --version | sed 's/.*\(v[0-9][^+]*\).*/\1/')"
+        echo "::set-output name=calico-version::$(cat /tmp/calico.yaml | grep --max-count=1 "calico/cni:v" | sed 's/.*calico\/cni:\(.*\)/\1/')"
+        echo "::set-output name=helm-version::$(helm version --short | sed 's/\([^+]*\).*/\1/')"
```

Comment:
Do you think we should remove the `v` prefix too? If anyone's going to parse and use these versions they'll have to remove it anyway.

Reply:
Hmmm... I don't think so. It may very well be that they want the version with v as well as without v if they want to automatically do something based on the version rather than just print it.

Given that I don't feel confident about it, I lean towards avoiding additional complexity.